Javascript/homework/responses/1.2-buttons.md

```diff
@@ -0,0 +1,15 @@
+### Week 1 - Printing the time using JS and HTML 
+
+*In this assignment, students will gain a thorough understanding of basic HTML syntax as well as JavaScript functions to print out the current time given some user input*
+
+#### Task 2: Getting user input with buttons.
+
+Next, lets set up a form for a user to select their timezone for the clock we're going to build.  To do this, we'll use radio buttons - these are the type of buttons that you'd associate with a multiple choice question - there are a few different options to select, but, since we only want to show one time, we only want the user to be able to select one of these buttons.
+
+Radio buttons can be defined using the HTML `input` field if you set the `type` attribute to `"radio"`.  In order to let the computer know that these items are in a group, be sure to give each input field the same same `name` attribute (I named mine `"timezone"`). Each item should also have a unique value tag.
```

Comment:
Do I need to search these items up on how to define things using the HTML input?